src/koala/utils/plots.py

```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8 -*-
-from __future__ import division
+from __future__ import absolute_import, division, print_function
 from builtins import str
-from builtins import range
```

Comment:
range is used in several places as `range(len(...))`, does this produce the same functionality in python 2.7 vs 3?

Reply:
The `builtins` module that `futurize` uses makes `range` behave like python 3 on python 2. As long as we don't need range to produce a list (and wrap it when we do), range is fine on either version.

Reply:
ah alright